src/filterFunction/categoryFilter.js

```diff
@@ -0,0 +1,22 @@
+export const categoryFiltered = (product,{Sports , Casual , Formal}) =>{
+    let categoryFilteredProduct = []; 
+
+    if (Sports === true)
```

Comment:
You can directly put here sports if it is holding boolean value.

Like:
If(Sports){ \\Do your stuff}

No need to do : sports === true

Reply:
i have not idea about this . thanks for explanation , will try it